native/package.json

```
@@ -41,7 +41,8 @@
     "lint:changed": "eslint . -c ../.eslintrc_changed.js",
     "lint:ci": "yarn run eslint --quiet --format junit -o reports/lint/junit-lint.xml",
     "eslint": "eslint .",
-    "postinstall": "jetifier && patch-package",
+    "license-crawler": "npm-license-crawler -onlyDirectDependencies -json ../assets/licenses_native.json",
```

Why do we save this to general assets directory if it is only relevant for native?
```suggestion
    "license-crawler": "npm-license-crawler -onlyDirectDependencies -json ./assets/licenses.json",
```